File: blog-dao/src/main/java/com/dao/NewsMapper.java
package com.dao;

import com.entity.News;

import java.util.List;

public interface NewsMapper {
    int deleteByPrimaryKey(Integer nid);

    int insert(News record);

    int insertSelective(News record);

    News selectByPrimaryKey(Integer nid);

    int updateByPrimaryKeySelective(News record);

    int updateByPrimaryKey(News record);

    //查询出所有的每日一句
    List<News> selectAllNews();

}

File: blog-dao/src/main/java/com/dao/NewsMapper.kt
package com.dao

import com.entity.News

interface NewsMapper {
    fun deleteByPrimaryKey(nid: Int?): Int

    fun insert(record: News): Int

    fun insertSelective(record: News): Int

    fun selectByPrimaryKey(nid: Int?): News

    fun updateByPrimaryKeySelective(record: News): Int

    fun updateByPrimaryKey(record: News): Int

    //查询出所有的每日一句
    fun selectAllNews(): List<News>

}